onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -1426,2032 +1440,2037 @@ KernelCreateInfo BuildKernelCreateInfo<void>() {
 
 Status RegisterOnnxOperatorKernels(KernelRegistry& kernel_registry) {
   static const BuildKernelCreateInfoFn function_table[] = {
-      BuildKernelCreateInfo<void>,  // default entry to avoid the list become empty after ops-reducing
```

Comment:
there are a lot of whitespace changes in this file. please ensure that it is formatted (the CI build will also complain about this).

you can use lintrunner. more details here:
https://github.com/microsoft/onnxruntime/blob/main/docs/Coding_Conventions_and_Standards.md#linting